src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -249,6 +251,12 @@ const validCustomText = ({ customText }) => {
   }
 };
 
+const validBenefitDescription = formConfig => {
+  validObjectProperty(formConfig, 'benefitDescription', true);
```

Comment:
This should only be required if SiP is enabled, but that's a future refactor, I think.